docs/ENUM.md

```diff
@@ -402,7 +402,14 @@ ENUM and Sprout
 ---------------
 To enable ENUM lookups on Sprout, edit `/etc/clearwater/user_settings` and add the following configuration to use either an ENUM server (recommended) or an ENUM file
 
-    enum_server=<hostname of enum server>
+    enum_server=<IP addresses of enum servers>
```

Comment:
Above this it says to put the enum config in user_settings. Is that correct? If so, can you move your new enum_server option (plus the other enum options) in the configuration guide? If not, can you fix this here?